lib/language_pack/helpers/bundler_wrapper.rb

```diff
@@ -37,7 +38,16 @@ def install
 
   def clean
     ENV['BUNDLE_GEMFILE'] = @orig_bundle_gemfile
-    FileUtils.remove_entry_secure(bundler_path) if Dir.exist?(bundler_path)
+    FileUtils.remove_entry_secure(@bundler_tmp) if Dir.exist?(@bundler_tmp)
+
+    if LanguagePack::Ruby::BUNDLER_VERSION  == "1.7.12"
+      # Hack to cleanup after pre 1.8 versions of bundler. See https://github.com/bundler/bundler/pull/3277/
+      Dir["#{Dir.tmpdir}/bundler*"].each do |dir|
+        FileUtils.remove_entry_secure(dir) if Dir.exist?(dir) && File.stat(dir).writable?
+      end
+    else
+      raise "Bundler cleanup hack is out of date, please update it to be compatible with #{LanguagePack::Ruby::BUNDLER_VERSION}"
```

Comment:
Ah if 1.9.6 is going out this week that is probably soon enough to not worry about this hack. Assuming we can get a new version of the buildpack within a week or two I'm ok just removing this whole if block since the desired behavior should be included in the new version of bundler.